File: emulinker/src/main/java/org/emulinker/kaillera/model/event/GameClosedEvent.java
package org.emulinker.kaillera.model.event;

import org.emulinker.kaillera.model.*;

public class GameClosedEvent implements ServerEvent {
  private KailleraServer server;
  private KailleraGame game;

  public GameClosedEvent(KailleraServer server, KailleraGame game) {
    this.server = server;
    this.game = game;
  }

  public String toString() {
    return "GameClosedEvent";
  }

  public KailleraServer getServer() {
    return server;
  }

  public KailleraGame getGame() {
    return game;
  }
}

File: emulinker/src/main/java/org/emulinker/kaillera/model/event/GameClosedEvent.kt
package org.emulinker.kaillera.model.event

import org.emulinker.kaillera.model.KailleraGame
import org.emulinker.kaillera.model.KailleraServer

class GameClosedEvent(override val server: KailleraServer, val game: KailleraGame) : ServerEvent {
  override fun toString(): String {
    return "GameClosedEvent"
  }
}
